### DISCUSSION TITLE
Syntax Error Updating content of BlogPost

### DISCUSSION THREAD

**Discussion by devhomecali** [OrchardCMS/OrchardCore#15997] [Q&A]
Following the guide https://docs.orchardcore.net/en/dev/docs/guides/decoupled-cms/#configuring-the-preview-feature-for-blog-posts. 

Following @deanmarcussen's suggestion to solve the problem of (https://github.com/OrchardCMS/OrchardCore/issues/6338), I installed **OrchardCore.Application.Cms.Tarjets package version v1.0.0-rc1-13380**.

But now Update generate syntax error. [See this video](https://github.com/devhomecali/OrchardCore/blob/master/OrchardCore-Error-Save-BlogPost.gif)

See another error (https://github.com/OrchardCMS/OrchardCore/issues/6348)

**Comment by ns8482e** [comment]
> Following the guide https://docs.orchardcore.net/en/dev/docs/guides/decoupled-cms/#configuring-the-preview-feature-for-blog-posts.
> 
> Following @deanmarcussen's suggestion to solve the problem of (#6338), I installed **OrchardCore.Application.Cms.Tarjets package version v1.0.0-rc1-13380**.
> 
> But now Update generate syntax error. [See this video](https://github.com/devhomecali/OrchardCore/blob/master/OrchardCore-Error-Save-BlogPost.gif)
> 
> See another error (#6348)

you can  only use `{{ }}` if it's valid expression. Markdown editor internally uses Liquid template to convert to `Html`
